compiler/rustc_trait_selection/src/traits/query/dropck_outlives.rs

```diff
@@ -146,14 +162,17 @@ pub fn compute_dropck_outlives_inner<'tcx>(
             result.overflows.len(),
             ty_stack.len()
         );
-        dtorck_constraint_for_ty_inner(
+        match dtorck_constraint_for_ty_inner(
             tcx,
             ocx.infcx.typing_env(param_env),
-            DUMMY_SP,
+            span,
             depth,
             ty,
             &mut constraints,
-        )?;
+        ) {
+            Err(_) => return Err(Vec::new()),
```

Comment:
Do we ever actually hit this path? I wonder if we should just delay a bug `dtorck_constraint_for_ty_inner` and return `Ok` -- seems to only return `NoSolution` for placeholder/infer/error.